wafer/talks/models.py

```diff
@@ -129,7 +129,8 @@ class Meta:
 
     notes = models.TextField(
         null=True, blank=True,
-        help_text=_("Any notes for the conference organisers?"))
+        help_text=_("Any notes for the conference organisers?  "
+                    "Those are not visible to the public."))
```

Comment:
I don't like the use of "Those" here - I would prefer "These are not visible ..."

Reply:
Fixed.